Library/Formula/memo.rb

```diff
@@ -0,0 +1,16 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.5.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "3e047b09e91d695f1767d1dd6d179732c07a5759"
+
+  def install
+    system "make"
```

Comment:
Creating the `bin` directory. `man1.mkpath` is not necessary as the Makefile creates it.